opentracing-spring-cloud-starter-jaeger/src/main/java/io/opentracing/contrib/spring/cloud/starter/jaeger/JaegerConfigurationProperties.java

```diff
@@ -0,0 +1,302 @@
+/**
+ * Copyright 2017-2018 The OpenTracing Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.opentracing.contrib.spring.cloud.starter.jaeger;
+
+import com.uber.jaeger.Configuration;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@ConfigurationProperties("opentracing.jaeger")
+public class JaegerConfigurationProperties {
+
+  private final RemoteReporterProperties remoteReporterProperties = new RemoteReporterProperties();
+  private final HttpSender httpSender = new HttpSender();
+  private final UdpSender udpSender = new UdpSender();
+  private final ConstSampler constSampler = new ConstSampler();
+  private final ProbabilisticSampler probabilisticSampler = new ProbabilisticSampler();
+  private final RateLimitingSampler rateLimitingSampler = new RateLimitingSampler();
+  private final RemoteControlledSampler remoteControlledSampler = new RemoteControlledSampler();
+  /**
+   * Enable Jaeger Tracer
+   */
+  private boolean enabled = true;
+  /**
+   * If this options is enabled, then the instantiation of the Tracer will be handed off to the
+   * {@link com.uber.jaeger.tracerresolver.JaegerTracerResolver} Enabling this option means that
+   * none of the autoconfigured beans apply. Furthermore the mere setting of this option to true is
+   * not enough for it to take effect, {@link com.uber.jaeger.tracerresolver.JaegerTracerResolver}
+   * also needs to be on the classpath
+   */
+  private boolean useTracerResolver = false;
+  /**
+   * The serviceName that the tracer will use
+   */
+  private String serviceName = "spring-boot";
```

Comment:
can we use `spring.application.name= # Application name.` instead of this?

Reply:
Done